notary-signing-service/src/routes/account.js

```diff
@@ -21,4 +22,46 @@ router.get('/', async (_req, res) => {
   });
 });
 
+/**
+ * @swagger
+ * /account/info:
+ *   post:
+ *     description: Signs specific notary info
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         description: When the notary info could be correctly signed
+ */
+router.post('/info', async (req, res) => {
+  const address = account.getAddress();
+  const publicKey = account.getPublicKey();
+  const {
+    name, notarizationUrl, dataResponsesUrl, headsUpUrl,
+  } = req.body;
+
+  const privateKey = account.getPrivateKey();
+
+  // you cannot spread an object in varargs
+  const signature = signPayload(
+    privateKey,
+    name,
+    address,
+    notarizationUrl,
+    dataResponsesUrl,
+    headsUpUrl,
+    publicKey,
+  );
```

Comment:
You can do it something like
```javascript
const signature = signPayload(
    privateKey,
    address,
    publicKey,
    ...req.body
  );
```
It should take you the parameters properly in `signPayload` 🤔, at least I used it like this before.

Reply:
you can't since `body` is non-iterable, therefore has no order and `varargs` do. Besides, doing that would require the user to send the request in the exact form that the API will use it to create the signature.